Data Files/MWSE/mods/CraftingFramework/types/Tool.lua

```diff
@@ -0,0 +1,14 @@
+---@meta
+
+---@class craftingFrameworkToolData
+---@field id string **Required.**  This will be the unique identifier used internally by Crafting Framework to identify this `tool`.
+---@field name string The name of the tool. Used in various UIs.
+---@field ids table<number, string> **Required.**  This is the list of game item ids that will be used as the tool.
+
+---@class craftingFrameworkToolRequirements
+---@field equipped boolean Perform a check whether the tool is equipped?
```

Comment:
When true, room must be equipped to be considered valid.

This table also has a count field which determines how many of the items need to be in the players inventory

Reply:
The full definition of thus requirements take can be found in Recipe.lua in the ToolRequirementsSchema

https://github.com/jhaakma/crafting-framework/blob/d0b62aace3a54de8bea2113f8cb8ea3522c3083a/Data%20Files/MWSE/mods/CraftingFramework/components/Recipe.lua#L17